models/notifications/requests/webhook_request.py

```diff
@@ -39,15 +32,40 @@ def send(self):
         """ Attempt to send the notification."""
         # Build the request
         headers = {
-            'Content-Type': 'application/json',
+            'Content-Type': 'application/json; charset="utf-8"',
             'X-TBA-Version': '{}'.format(WEBHOOK_VERSION)
         }
-        message_json = self._json_string()
+        payload = self._json_string()
         # Generate checksum
-        headers['X-TBA-Checksum'] = self._generate_webhook_checksum(message_json)
-
-        from google.appengine.api import urlfetch
-        return urlfetch.fetch(url=self.url, payload=message_json, method=urlfetch.POST, headers=headers)
+        headers['X-TBA-Checksum'] = self._generate_webhook_checksum(payload)
+
+        import logging, urllib2
+        request = urllib2.Request(self.url, payload, headers)
+
+        # TODO: Consider more useful way to surface error messages
+        # https://github.com/the-blue-alliance/the-blue-alliance/issues/2576
+        valid_url = True
+        try:
+            urllib2.urlopen(request)
+        except urllib2.HTTPError, e:
```

Comment:
This adds back in Phil's old code to handle unexpected responses from webhooks. I hate it, but my code is problematic, so let's do this instead